notebooks/recommendation_systems/solutions/tfrs_basic_retrieval.ipynb

```diff
@@ -0,0 +1,1364 @@
+{
```

Comment:
<p>I don't think there has been an explanation of what "two-tower" model is at this point yet.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Added an explanation at the beginning of the notebook.</p>